resources/views/backend/event/create.blade.php

```diff
@@ -65,6 +65,76 @@
+                                    <li class="dropdown-item">
+                                        <label class="form-check-label d-block w-100" for="event_type_{{ $key }}">
+                                            <input class="form-check-input me-2 event-type-checkbox" type="checkbox" name="event_type[]" value="{{ $key }}" id="event_type_{{ $key }}" data-label="{{ $value }}">
+                                            {{ $value }}
+                                        </label>
+                                    </li>
+                                @endforeach
+                            </ul>
+                        </div>
+                        @error('event_type')
+                            <strong class="text-danger">{{ $message }}</strong>
+                        @enderror
+                    </div>
+
+                    <!-- Selected tags appear here -->
+                    <div class="col-md-5">
+                        <div id="selected-tags" class="mt-2"></div>
+                    </div>
+                </div>
+
+                <script>
+                    document.addEventListener('DOMContentLoaded', function() {
+                        const checkboxes = document.querySelectorAll('.event-type-checkbox');
+                        const selectedTagsContainer = document.getElementById('selected-tags');
+
+                        checkboxes.forEach(function(checkbox) {
+                            checkbox.addEventListener('change', function() {
+                                updateTags();
+                            });
+                        });
+
+                        function updateTags() {
+                            selectedTagsContainer.innerHTML = ''; // Clear previous tags
+                            checkboxes.forEach(function(checkbox) {
+                                if (checkbox.checked) {
+                                    const tagLabel = checkbox.getAttribute('data-label');
+                                    const tagElement = document.createElement('span');
+                                    tagElement.classList.add('badge', 'bg-primary', 'me-1', 'mb-1');
+                                    tagElement.innerHTML = tagLabel + ' <i class="bi bi-x-circle ms-1" style="cursor:pointer;"></i>'; // 'x' icon for removal
+                                    
+                                    // Remove tag when clicked
+                                    tagElement.querySelector('i').addEventListener('click', function() {
+                                        checkbox.checked = false;
+                                        updateTags();
+                                    });
+
+                                    selectedTagsContainer.appendChild(tagElement);
+                                }
+                            });
+                        }
+                    });
+                </script>
+
+
+
+
+
+
+
```

Comment:
Can you try something like this? It is simple and robust

https://dev.to/snehalkadwe/how-to-use-select-2-with-multiple-options-in-laravel-and-livewire-4p4a (Hope you can do alphine)

Or you can reuse the item adder with some small changes.